docs/gke/configs/deploy.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
`set -e` might be an issue because `kubectl create namespace` can fail if namespace is already created by bootstrapper.

Reply:
I did see that comment in the other script but it appears to be working even with setting this flag. I can add an `OR` flag after this command.

Reply:
I add an `unset -e` just before the namespace and secrets creation towards the end of the script.